meson.build

```diff
@@ -71,6 +71,9 @@ conf.set_quoted('PREVIEWS_DIR', join_paths(libdir, PREVIEWS))
 WSWITCHER = 'budgie-wswitcher'
 conf.set_quoted('WSWITCHER_DIR', join_paths(PLUGINS_INSTALL_DIR, WSWITCHER))
 
+WEATHERSHOW = 'budgie-weathershow'
+conf.set_quoted('WEATHERSHOW_DIR', join_paths(PLUGINS_INSTALL_DIR, WEATHERSHOW))
```

Comment:
hmm - well I suppose since we are fixing things, the /usr/lib area is for source code. /usr/share is for data.  icons is really data so instead of PLUGS_INSTALL_DIR we could use PLUGINS_DATA_DIR.  Thoughts

Reply:
Definitely makes sense to me,  I will rework it to handle it this way.